profiles/targets/clang/package.use.force

```diff
@@ -0,0 +1,14 @@
+# Copyright 1999-2016 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License, v2
+# $Id$
+
+# Force libc++ as the default stdlib, and compiler-rt as the default rtlib.
+sys-devel/llvm clang default-libcxx default-compiler-rt
+
+# Build libc++ and libcxxrt against libunwind instead of libgcc_s.
+sys-libs/libcxx libunwind
+sys-libs/libcxxrt libunwind
+
+# This flag frees iproute2 from depending on libmnl, which is mis-compiled by
+# clang.
+sys-apps/iproute2 minimal
```

Comment:
you need to solve your ~amd64 by stabilizing the correct packages in the tree.  there's no way we can make this entire profile ~arch and not get a mess.  if you can't stabilize the packages in the tree, we can put them on the musl overlay as stable for now.  eventually that entire overlay should be migrated into the tree, but that's still a long way off.